apps/openmw/mwrender/objects.hpp

```diff
@@ -6,7 +6,8 @@
 
 #include <osg/ref_ptr>
 #include <osg/Object>
-
+#include <osg/Vec3>
```

Comment:
This specific header doesn't use vectors in its declarations. Put this include into files that actually use vectors to keep compilation times shorter.

Reply:
which file do you think about?

Reply:
Um. The source files that fail to compile if the line is removed.